schemactl/deploy.py

```diff
@@ -0,0 +1,20 @@
+import os
+import util
+import requests
+
+# Submit the ecosystem schema file to the API.
+def deploy(file):
+    if not os.path.isfile(file):
+        raise Exception(f"The file {file} does not exist")
+
+    api_key = util.get_env_var("DEPLOY_API_KEY")
+    api_url = util.get_env_var("DEPLOY_API_URL")
+
+    body = open(file).read()
```

Comment:
Could use `with open` to not leave the file open while the rest of the function call executes.